src/Common/PathId_test.cpp

```diff
@@ -0,0 +1,97 @@
+/*
+ Copyright 2021 Alain Dargelas
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+#include <Surelog/Common/FileSystem.h>
+#include <Surelog/Common/PathId.h>
+#include <Surelog/SourceCompile/SymbolTable.h>
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include <filesystem>
+#include <memory>
+
+namespace SURELOG {
+
+using ::testing::ElementsAre;
+namespace fs = std::filesystem;
+
+namespace {
+class TestFileSystem : public FileSystem {
+ public:
+  TestFileSystem() : FileSystem(std::filesystem::current_path()) {}
+};
+
+TEST(PathId, constructors) {
+  std::unique_ptr<SymbolTable> symbolTableA(new SymbolTable);
```

Comment:
For readability, I suggest to have `new` calls on constructors look like function calls: `new SymbolTable()`